packages/public/server/src/module/Ui/templates/user/user/register.twig

```diff
@@ -72,6 +72,11 @@
             {{ passwordConfirmErrors }}
         </div>
     </div>
+    <div class="form-group{{ interestsErrors ? ' has-error' }}">
```

Comment:
this line looks weird. Is this fine w/o an else case?

Reply:
Hmm, doesn't look correct, but it was like that before... I think I'll just add the else case, but seems to work o.O

Reply:
Well, actually the whole file is like this, and it's supported since twig 1.12.0 according to https://stackoverflow.com/a/24859759 while we're on 1.20.0 so I think this is fine.

Reply:
And ppl say that JS is weird xD